replacements.js

```diff
@@ -1,6 +1,7 @@
 const replacements = [
 	// German umlauts
 	['ß', 'ss'],
+	['ẞ', 'Ss'],
```

Comment:
This is unrelated and should be a separate pull request.

Reply:
sry, created an extra pr for this.